eq-author-api/src/validation/customKeywords/validateDuration.js

```diff
@@ -0,0 +1,29 @@
+const {
+  ERR_EARLIEST_AFTER_LATEST,
+} = require("../../../constants/validationErrorCodes");
+
+module.exports = function(ajv) {
+  ajv.addKeyword("validateDuration", {
+    $data: true,
+    validate: function isValid(otherFields, entityData, fieldValue, dataPath) {
+      isValid.errors = [];
+
+      const valid = true;
+
+      if (!valid) {
+        isValid.errors = [
+          {
+            keyword: "errorMessage",
+            dataPath,
+            message: ERR_EARLIEST_AFTER_LATEST,
+            params: {},
+          },
+        ];
+
+        return false;
+      }
+
+      return true;
+    },
```

Comment:
Haha - Matt, no probs - your head is fine and you're totally correct - Since we split this ticket after I had done a lot of work on duration validation. this is simply to ensure duration validation doesn't break in the meantime. There is no actual validation occuring here so we set it to true until the code gets written.
I hope to follow up this ticket immediately with the new duration Validation ticket and continue off from here - not the tidiest solution, but saves me removing loads of graphql stuff i'd already done